src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/ExpectedOutputs/OneResultWithTwoTraces.sarif

```diff
@@ -157,7 +147,9 @@
                           "text": "myReader no longer refers to a database resource"
                         }
                       },
-                      "kind": "EndScope"
+                      "kinds": [
+                        "EndScope"
```

Comment:
>EndScope [](start = 25, length = 8)

Although `kinds` is a free string, still it would be nice if some of the examples matched the recommended strings in the schema description of `threadFlowLocation.kinds`. Here, for example:
```
"kinds": [
  "scope",
  "exit"
]
```

Reply:
Good suggestion, yes.

Reply:
This requires a converter change to handle the conversion to SARIF kinds. Deferred. 

[https://github.com/Microsoft/sarif-sdk/issues/1293](https://github.com/Microsoft/sarif-sdk/issues/1293)

---
In reply to: [257516679](https://github.com/Microsoft/sarif-sdk/pull/1292#discussion_r257516679) [](ancestors = 257516679)